playbooks/lambda/create.yml

```diff
@@ -10,6 +10,9 @@
         role: "{{ aws_role }}"
         handler: image_resize.lambda_handler
       register: lambda_info
+      # If a new aws_role is created there is 5-6sec delay before lambda can assume the role
+      until: lambda_info is succeeded
+      delay: 6

```

Comment:
This is not relevant only for `undeploy` therefore it should be added as a separate pull request.